wg-manifests/charter.md

```diff
@@ -0,0 +1,100 @@
+# WG Manifests Charter
+
+This charter adheres to the conventions, roles and organization management
+outlined in [wg-governance].
+
+## Scope
+
+- Provide a catalog (centralized repository) of Kubeflow application manifests.
+- Provide a catalog of third-party apps for common services.
+- Provide documentation to help users install a set, or all of the included
+  apps.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Maintain a set of manifests that will allow users to install Kubeflow apps and
+  common services easily on Kubernetes, either on the cloud or on-prem, without
+  depending on external cloud services or closed source solutions. These
+  manifests will be developed with certain principles in mind, which are
+  outlined in the last bullet. Those include:
+    1. A common set of manifests for the current five (5) official Kubeflow
+       applications:
+        - Training Operators
+        - Kubeflow Pipelines (KFP)
+        - Notebooks
+        - KFServing
+        - Katib
+    1. Manifests for a set of specific common services:
+       - Istio
+       - KNative
+       - Dex
+       - Cert-Manager
+    1. Manifests for common, currently unmaintained, apps:
```

Comment:
Why are you distinguishing between "unmaintained" and "maintained" apps? If your just copying the manifests from the upstream repo to this repo then isn't it the same?

Reply:
Will reword